### DISCUSSION TITLE
Call for maintainers

### DISCUSSION THREAD

**Discussion by bobheadxi** [bobheadxi/deployments#103] [Announcements]
Hey everyone, there are a number of [open issues](https://github.com/bobheadxi/deployments/issues/92) and [discussions](https://github.com/bobheadxi/deployments/discussions/categories/q-a) for this project but unfortunately, I haven't been able to take a look at this and probably won't have the bandwidth to actively maintain this project for the time being. If anyone is open to helping me maintain this project I'd be super grateful and happy to chat, please reach out to me via [twitter](https://twitter.com/bobheadxi), email, or in this discussion!

**Comment by vhatsura** [comment]
Hey, @bobheadxi 
I'm here and eager to help you maintain this awesome action

**Reply by bobheadxi** [reply]
Hey @vhatsura , thank you for offering! Do you mind performing a review https://github.com/bobheadxi/deployments/pull/104 and helping to guide it to merge? :)

**Reply by vhatsura** [reply]
@bobheadxi, the PR was fixed, but my approval doesn't allow me to merge it

**Reply by farioas** [reply]
Hi @bobheadxi !
Just pinging you to get these things done

**Reply by bobheadxi** [reply]
I've invited you as a collaborator @vhatsura

**Reply by buckett** [reply]
I'm happy to help if I can @bobheadxi / @vhatsura

**Reply by vhatsura** [reply]
Hey, @bobheadxi 

Unfortunately, a lot was changed, and I can not provide enough time for this project. Could you please remove me from collaborators and add @buckett? He will be much better than me.

**Reply by bobheadxi** [reply]
Updated! Thanks for the heads up.

**Comment by aismailov** [comment]
Hi,
I like your work here, it is quite useful. Happy to contribute to the repo.
